### PR TITLE
add a `k` argument to random commands

### PR DESCRIPTION
**Pull request by selaere**
i pasted this into vscode and there aren't any warnings 
i hope this isn't full of bugs

### CODE REVIEW

File: cn2/__main__.py [hunk 1]
```diff
@@ -121,17 +121,23 @@ async def all_(ctx):
         out.append(f"{playername}: WP = {wp}")
     await ctx.send("\n".join(out))
 
-@bot.command(name="random")
-async def random_(ctx, a: Optional[int] = 10, b: int = 1):
+@bot.command()
+async def random(ctx, a: int = 10, b: int = 1, k: int = 1):
```
Comment (ubq323) [inline cn2/__main__.py:125]: why have you changed this back

**Review by ubq323 — CHANGES_REQUESTED** [review]

File: cn2/__main__.py (hunk 1 above)
Comment (selaere) [reply cn2/__main__.py:125]: we aren't importing random anyway
and Optional[] is not needed

File: cn2/__main__.py [hunk 2]
```diff
@@ -121,17 +121,23 @@ async def all_(ctx):
...
+        return
     if b < a:
         a, b = b, a
-    await ctx.send(f"I choose **{randint(a, b)}**")
+    await ctx.send(f"I choose **{', '.join(randint(a, b) for _ in range(k))}**")
 		   
-@bot.command()
-async def random_emoji(ctx):
+@bot.command(aliases = ["emoji"])
+async def random_emoji(ctx, k: int = 10):
```
Comment (ubq323) [inline cn2/__main__.py]: shouldn't the default be 1

**Review by ubq323 — CHANGES_REQUESTED** [review]

File: cn2/__main__.py (hunk 2 above)
Comment (selaere) [reply cn2/__main__.py]: done

**Pull request closed without merging**